app/models/katello/glue/candlepin/pool.rb

```diff
@@ -76,6 +76,7 @@ def organization
       # to define instance variables
       # TODO: break up method
       # rubocop:disable MethodLength
+      # rubocop:disable CyclomaticComplexity
       def load_remote_data(attrs)
```

Comment:
http://projects.theforeman.org/issues/9486

Large enough to do in separate PR for future release